mods/d2k/maps/atreides-01a/map.yaml

```diff
@@ -134,6 +134,8 @@ Rules:
 		-ConquestVictoryConditions:
 		MissionObjectives:
 			EarlyGameOver: true
+		Production:
+			Produces: Building
```

Comment:
Is this still necessary? Selling my ConYard has an odd side-effect.
While that's not an issue, you need to just exclude Upgrades here, not move Building to the player, right?